pytorch_lightning/trainer/trainer.py

```diff
@@ -1085,11 +1085,12 @@ def tune(
     def _restore_modules_and_callbacks(self, checkpoint_path: Optional[_PATH] = None) -> None:
         # restore modules after setup
         self.checkpoint_connector.resume_start(checkpoint_path)
-        self.checkpoint_connector.restore_model()
-        self.checkpoint_connector.restore_datamodule()
+        # FIXME
```

Comment:
@awaelchli @carmocca so the suggestion here was to add a protected method on `checkpoint_connector` for callback restore which would be used only by this QAT callback, right?